cpp/INIReader.h

```diff
@@ -66,6 +66,10 @@ class INIReader
     // not found or not a valid integer (decimal "1234", "-1234", or hex "0x4d2").
     INI_API long GetInteger(const std::string& section, const std::string& name, long default_value) const;
 
+    // Get an unsigned integer (unsigned long) value from INI file, returning default_value if
+    // not found or not a valid integer (decimal "1234", or hex "0x4d2").
```

Comment:
Let's wrap these comment lines at <80 chars, so after "returning" and after "or".